internal/descriptor/grpc_api_configuration_test.go

```diff
@@ -67,8 +67,8 @@ http:
 		t.Fatal(err)
 	}
 
-	if !strings.Contains(err.Error(), "line 7") {
-		t.Errorf("Expected yaml error to be detected in line 7. Got other error: %v", err)
+	if !strings.Contains(err.Error(), "line 6") {
```

Comment:
Do we know why this changed?

Reply:
The error message changed to 

```
yaml: line 6: did not find expected key
```

Reply:
Haha, yeah, I can kind of infer that, I was just curious why the line number had changed from 7 to 6.